src/routes/index.ts

```diff
@@ -0,0 +1,69 @@
+import { Hono } from 'hono';
+import { user } from '@/controllers/userController';
+import { DBError } from '@/validation/custom-error';
+import { errorCode } from '@/utils/utils';
+import { BodySchema, formatZodError } from '@/validation/body-check';
+
+export const userRoutes = () => {
+  const app = new Hono();
+
+  app.onError((err, c) => {
+    if (err instanceof DBError) {
+      switch (err.code) {
+        case errorCode.INVALID: {
+          c.status(400);
+          break;
+        }
+        case errorCode.CONNECTION: {
+          c.status(502);
+          break;
+        }
+        case errorCode.INTERNAL_SERVER_ERROR: {
+          c.status(500);
+          break;
+        }
+        case errorCode.NOT_FOUND: {
+          c.status(404);
+          break;
+        }
+      }
+    }
+    return c.json(err.message);
+  });
+  app.get('/:id', async (c) => {
```

Comment:
In a route file the routes should look like something like this
`app.get('/:id', middleware, middleware, ...,user.findOne);`